src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,838 @@
+ :on-boarding/section-journals "Notas diárias"
+ :on-boarding/section-pages "PÁGINAS"
+:on-boarding/section-phone "telefone"
+:on-boarding/section-app "Aplicativo interno"
+:on-boarding/section-config "Arquivo de configuração"
+:query/config-property-settings "Configurações de propriedades para esta consulta:"
+:bug-report/main-title "Relatório de bug"
+:bug-report/clipboard-inspector-title "Inspetor de dados da área de transferência"
+:bug-report/main-desc "Você pode nos ajudar enviando um relatório de bug? Vamos resolver isso o mais rápido possível."
+:bug-report/section-clipboard-title "O bug que você encontrou está relacionado a esses recursos?"
+:bug-report/section-clipboard-desc "Você pode usar essas ferramentas úteis para nos fornecer informações adicionais."
+:bug-report/section-clipboard-btn-title "Auxiliar da área de transferência"
+:bug-report/section-clipboard-btn-desc "Inspetar e coletar dados da área de transferência"
+:bug-report/section-issues-title "Ou..."
+:bug-report/section-issues-desc "Se não houver ferramentas disponíveis para coletar informações adicionais, relate o bug diretamente."
+:bug-report/section-issues-btn-title "Enviar um relatório de bug"
+:bug-report/section-issues-btn-desc "Ajude a melhorar o Logseq!"
+:bug-report/inspector-page-desc-1 "Pressione Ctrl+V / ⌘+V para inspecionar os dados da área de transferência"
+:bug-report/inspector-page-desc-2 "ou clique aqui para colar se estiver usando a versão mobile"
+:bug-report/inspector-page-placeholder "Pressione e segure aqui para colar se estiver no mobile"
+:bug-report/inspector-page-tip "Algum problema? Sem problemas, clique para voltar para o passo anterior."
+:bug-report/inspector-page-btn-back "Voltar"
+:bug-report/inspector-page-btn-copy "Copiar o resultado"
+:bug-report/inspector-page-copy-notif "Copiado para a área de transferência!"
+:bug-report/inspector-page-btn-create-issue "Criar uma issue"
+:bug-report/inspector-page-desc-clipboard "Aqui estão os dados lidos da área de transferência."
+:bug-report/inspector-page-desc-copy "Se for seguro compartilhar, clique no botão de cópia."
+:bug-report/inspector-page-desc-create-issue "Agora você pode relatar o resultado colado na sua área de transferência. Cole o resultado na seção 'Contexto Adicional' e indique de onde você copiou o conteúdo original. Obrigado!"
+:help/title-usage "Uso"
+:help/title-community "Comunidade"
+:help/title-development "Desenvolvimento"
+:help/title-about "Sobre"
+:help/title-terms "Termos"
+:help/start "Começando"
+:help/about "Sobre o Logseq"
+:help/roadmap "Roteiro"
+:help/bug "Relatório de bug"
+:help/feature "Solicitação de recurso"
+:help/changelog "Registro de alterações"
+:help/blog "Blog do Logseq"
+:help/docs "Documentação"
+:help/privacy "Política de privacidade"
+:help/terms "Termos"
+:help/forum-community "Comunidade do Fórum"
+:help/awesome-logseq "Awesome Logseq"
+:help/shortcuts "Atalhos de teclado"
+ :help/shortcuts-triggers "Triggers"
```

Comment:
We can use the equivalent in Portuguese, "Gatilhos". I've seen it used in other Open Source projects.

Reply:
In fact, from this line up to 170, the content is still in English. In order not to make future translation work difficult, we can remove these lines or translate them, okay?

Reply:
I corrected the translation of the mentioned lines.